conda/pytorch-nightly/bld.bat

```diff
@@ -11,6 +11,8 @@ if "%USE_CUDA%" == "0" (
 ) else (
     set build_with_cuda=1
     set desired_cuda=%CUDA_VERSION%
+    :: Set up nodot version for use with magma
+    set desired_cuda_nodot=%CUDA_VERSION:.=%
```

Comment:
cc @peterjc123 just to check that my batch scripting is correct here

Reply:
stolen from, https://www.tutorialspoint.com/batch_script/batch_script_replace_string.htm

Reply:
@seemethere This is correct.